rs/embedders/benches/heap.rs

```diff
@@ -131,10 +161,23 @@ fn heap_canister_init_body(mem: Mem, size: Size, src: Src) -> String {
     }
 }
 
+fn heap_canister_setup_body(mem: Mem, size: Size, src: Src) -> String {
+    let op = heap_op(Op::Write, mem);
+    let loop_body = loop_body(&op, Dir::Fwd, PAGE_SIZE, size, Step::TwoPages);
+    match src {
+        Src::Mix => format!(
+            r#"
+                {loop_body}
+            "#
+        ),
+        Src::Checkpoint | Src::PageDelta | Src::NewAllocation => String::new(),
```

Comment:
Why do we set this to a no-op instead of not calling the setup function in those cases?

Reply:
The setup is for the `mixed` scenarios, where some pages come from the checkpoint file and some from the page delta.

Reply:
I'll add some comments to clarify...

Reply:
> This function is executed once right before the `mixed` benchmarks

This sounds like this line with `String::new()` is unreachable so why not just return
```
format!(
            r#"
                {loop_body}
            "#
        )
```
from this function unconditionally?

Reply:
Then could you please fix this comment: "This function is executed once right before the `mixed` benchmarks"